app/tag/tag-controller.js

```diff
@@ -20,11 +20,18 @@ angular.module('tag-controller', ['registry-services'])
     $scope.repositoryName = $route.current.params.repositoryName;
     $scope.repository = $scope.repositoryUser + '/' + $scope.repositoryName;
     $scope.tagName = $route.current.params.tagName;
-
-    // How to query the tags
+    $scope.tagsDetails = [];
```

Comment:
This `tagsDetails` looked suspicious to me since you fill the tags with all details below. By grepping through the code of yours I found that is not used any longer. There fore I suggest to remove this line.